keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/ArtifactVersionDeploying.kt

```diff
@@ -0,0 +1,10 @@
+package com.netflix.spinnaker.keel.events
+
+/**
+ * An event fired when we launch a task to deploy a new version to a cluster
+ */
+data class ArtifactVersionDeploying(
+  val resourceId: String,
+  val artifactVersion: String,
```

Comment:
Would it be useful to have a reference to the `DeliveryArtifact` here, even if only for informational purposes (e.g. logging)?

Reply:
No, we need to look it up in the handler for this event because that's where we have access to the delivery config repository.